app/controllers/catalog_controller.rb

```diff
@@ -372,4 +374,18 @@ def get_leafdir(directory)
   def find_desc_metadata_file(job_output_directory)
     File.join(job_output_directory, bulk_job_metadata(job_output_directory)['argo.bulk_metadata.bulk_log_xml_filename'])
   end
+
+
+  # Sorts the Blacklight collection actions buttons so that the "Bulk Action" and "Bulk Update View" buttons appear
+  # at the front of the list.
+  def sort_collection_actions_buttons
+    collection_actions_order = blacklight_config.index.collection_actions.keys
+    collection_actions_order.delete(:bulk_update_view_button)
+    collection_actions_order.insert(0, :bulk_update_view_button)
+
+    # Use the order of indices in the collection_actions_order array for the Blacklight hash
+    blacklight_config.index.collection_actions = blacklight_config.index.collection_actions.to_h.sort do |(key1, value1), (key2, value2)|
```

Comment:
Unused block argument - `value1`. If it's necessary, use `_` or `_value1` as an argument name to indicate that it won't be used.<br>Unused block argument - `value2`. If it's necessary, use `_` or `_value2` as an argument name to indicate that it won't be used.